.github/workflows/kernel-demos.yml

```diff
@@ -161,20 +161,6 @@ jobs:
         working-directory: FreeRTOS/Demo/CORTEX_LM3S102_GCC
         run: make -j
 
-      - name: Build CORTEX_M3_MPS2_QEMU_GCC Demo
```

Comment:
Why do you remove these QEMU demo in yml file?

Reply:
The QEMU demo is removed from the mainline as well, so to pass the CI checks , it has been removed from this branch.